flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/rel/LogicalWindowAggregate.scala

```diff
@@ -18,8 +18,6 @@
 
 package org.apache.flink.table.plan.logical.rel
 
-import java.util
```

Comment:
Your current package is `org.apache.flink.table.plan.logical.rel`, in this package there is a` org.apache.flink.table.plan.logical.rel.util`, this and `util .List` package conflicts. So you can try the following 2 way solution:
solution1:
```
import java.util.List
groupSets: List[ImmutableBitSet],
aggCalls: List[AggregateCall])
```
solution2:
```
import java.{util => JUtil}
groupSets: JUtil.List[ImmutableBitSet],
aggCalls: JUtil.List[AggregateCall])
```